crates/ekore/src/anomalous_dimensions/unpolarized/spacelike/as4.rs

```diff
@@ -0,0 +1,58 @@
+//! |N3LO| |QCD|.
+
+pub mod ggg;
+pub mod ggq;
+pub mod gnsm;
+pub mod gnsp;
+pub mod gnsv;
+pub mod gps;
+pub mod gqg;
+
+use crate::harmonics::cache::Cache;
+use num::complex::Complex;
+
+// Compute the singlet anomalous dimension matrix.
```

Comment:
1. please take a look to `poe rdocs` + `poe rdocs-view` to see if things are explained well in the documentation
2. doing so, you will notice that to annotate functions in rust you need `///` as `//` is just for comments
3. since we are doing an iteration any ways, let's improve the docs: e.g. I think it would be good to spell explicitly the order of variations here `gg,gq,...`
4. for adding new refences to rust: add them to the (relevant) bib file and run `poe rbib`

Reply:
I can't run `poe rbib`, do you know how to fix this? 

```
(eko_dev)  pip list | grep bibtexparser
bibtexparser                    1.4.3
(eko_dev) poe rbib                      
Poe => python crates/make_bib.py > crates/ekore/src/bib.rs
Traceback (most recent call last):
  File "/Volumes/Git_Workspace/physicstools/NN3PDF/eko/crates/make_bib.py", line 45, in <module>
    bib_database = bibtexparser.parse_string(bib_str)
AttributeError: module 'bibtexparser' has no attribute 'parse_string'
```

Reply:
yes: a7760553 - sorry I was lazy :see_no_evil:

Reply:
Thanks

Reply:
> since we are doing an iteration any ways, let's improve the docs: e.g. I think it would be good to spell explicitly the order of variations here gg,gq,...

can you please take care of this?

Reply:
https://github.com/NNPDF/eko/blob/a72d78cd19782476d1766ab31765911b979b0eb8/crates/ekore/src/anomalous_dimensions/unpolarized/spacelike/as4.rs#L24-L26

Reply:
sorry I'm blind :see_no_evil: